tensorflow/lite/kernels/pooling_test.cc

```diff
@@ -26,8 +26,22 @@ using ::testing::ElementsAreArray;
 
 class BasePoolingOpModel : public SingleOpModel {
  public:
-  // TODO(ahentz): Also test different activation types, bias, padding types,
-  // stride values.
+  BasePoolingOpModel(const BuiltinOperator& type, const TensorData& input,
+                     int filter_width, int filter_height,
+                     const Padding& padding,
+                     const ActivationFunctionType& fused_activation_function,
+                     const TensorData& output) {
+    input_ = AddInput(input);
+    output_ = AddOutput(output);
+
+    SetBuiltinOp(type, BuiltinOptions_Pool2DOptions,
+                 CreatePool2DOptions(builder_, padding, 2, 2, filter_width,
```

Comment:
It seems that strides are hard-coded here. Could you make it configurable and make a unit test on strides also?

Reply:
@haozha111 , thanks for the feedback, i have updated the code as per your suggestion. Kindly check and approve.